library/globals.inc.php

```diff
@@ -259,7 +259,7 @@
       xl('Online Support Link'),
       'text',                           // data type
       'http://open-emr.org/',
```

Comment:
I presume our website for now... but www.libreehr.org is broken 404 not found ....
could temp set it to http://libreehr.github.io